packages/styled-components/src/constructors/styled.js

```diff
@@ -8,8 +8,8 @@ import type { Target } from '../types';
 const styled = (tag: Target) => constructWithOptions(StyledComponent, tag);
 
 // Shorthands for all valid HTML Elements
-domElements.forEach(domElement => {
-  styled[domElement] = styled(domElement);
-});
+for (let i = 0; i < domElements.length; i += 1) {
+  styled[domElements[i]] = styled(domElements[i]);
+}
```

Comment:
@anatoliy8493 if you use a for loop you need to use `hasOwnProperty` to check that it belongs to the object and not the prototype. forEach is better here imo.

Reply:
nope, it's needed in `for (var key in obj)`, not in `for | forEach`